lib/spryte/rspec/macros.rb

```diff
@@ -12,7 +12,8 @@ def request(name)
 
         host ENV.fetch("SPRYTE_RSPEC_HOST", "localhost")
 
-        let(:method) { :get }
+        let(:through) { :get }
+        let(:method) { method(:get) }
```

Comment:
I misunderstood - I thought you were suggesting replacing 

```ruby
let(:method) { :get }
```
with
```ruby
let(:method) { through(:get) }
```

And I couldn't see why that would help, since it wouldn't set `method` OR show a deprecation